examples/recording_service/pluggable_storage/c/README.md

```diff
@@ -102,7 +102,7 @@ Then you can call CMake like this:
 
 ```bash
 cmake -DCONNEXTDDS_DIR=<connext dir> -DCMAKE_TOOLCHAIN_FILE=<toolchain file created above>
-        -DCONNEXTDDS_ARCH=<connext architecture> ..
+      -DCMAKE_BUILD_TYPE=Release -DCONNEXTDDS_ARCH=<connext architecture> ..
```

Comment:
You should add -DBUILD_SHARED_LIBS=ON here. It is in the non-crosscompiled section.

Reply:
Will add.